src/Sylius/Bundle/CoreBundle/Controller/OrderController.php

```diff
@@ -113,8 +113,8 @@ public function afterPayAction(Request $request)
         $payment = $status->getFirstModel();
         $order = $payment->getOrder();
 
-        $orderStateResolver = $this->getOrderStateResolver();
-        $orderStateResolver->resolve($order);
+        $this->getOrderPaymentStateResolver()->resolve($order);
+        $this->getOrderShippingStateResolver()->resolve($order);
```

Comment:
Actually these resolvers should be be removed as they are not needed here (we use state machine, so they should be called when payment state is changed).

PS. Should be removed from docs example also